main.tf

```diff
@@ -34,6 +33,7 @@ module "orca" {
 
   ## OPTIONAL
   database_port                                        = var.database_port
+  default_multipart_chunksize_mb = var.default_multipart_chunksize_mb
```

Comment:
maybe add some spacing to the formatting

Reply:
Fixed